src/Features/Navigation/CoreMenu.php

```diff
@@ -181,7 +196,18 @@ public static function get_items() {
 				$order_items['all'],
 				$order_items['new'],
 				$product_items['all'],
-				$product_items['new'],
+				$product_cat_items['default'],
+				$product_tag_items['default'],
+				array(
+					'id'              => 'woocommerce-product-attributes',
+					'title'           => __( 'Attributes', 'woocommerce-admin' ),
+					'url'             => 'edit.php?post_type=product&page=product_attributes',
+					'capability'      => 'manage_product_terms',
+					'order'           => 40,
+					'parent'          => 'woocommerce-products',
+					'matchExpression' => 'edit.php(?=.*[?|&]page=product_attributes(&|$|#))|edit-tags.php(?=.*[?|&]taxonomy=pa_)(?=.*[?|&]post_type=product(&|$|#))',
```

Comment:
What do you think about documenting these expressions inline? Like: https://github.com/WordPress/WordPress/blob/master/wp-includes/formatting.php#L4552-L4562

Reply:
Hadn't seen this before; I like the idea.

Reply:
Updated to break into commented groups.